scripts/apeldbloader-cloud

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+#
+# Startup script for cloud APEL cloud
```

Comment:
"cloud APEL cloud"?

Could also do with bit more of a readme including an explanation of why this is similar to `dbld`.